_sass/minima/fastpages-styles.scss

```diff
@@ -196,22 +196,14 @@ h6:hover .anchor-link {
   font-weight:bold;
 }
 
-// Handle Overflow With Table Output
-
 table {
   display: block !important;
-  overflow-x: auto;
   white-space: nowrap;
   font-size: 75%;
   border:none;
   th{
     text-align: center! important;
   }
-  td{
-    text-overflow:ellipsis;
-    overflow:hidden;
-    max-width: 15em;
```

Comment:
I’m thinking 

1. we just bump this up a bit 
2. We add instructions in the README explaining what this is.

Reply:
that could work too, i'll quickly test it on my site with something really long too see what it actually does. i'd have thought it would get truncated at some point anyway when the df prints out but i'll do my best to break it and see.

Reply:
you're right, that was horrible.

have a look at this:
https://joedockrill.github.io/blog/misc/2020/08/16/Test-table-changes.html

that's with 
```
table {
  display: block !important;
  overflow-x: auto;
//  white-space: nowrap;
  font-size: 75%;
  border:none;
  max-width: 100%;
}
```
and nothing on td at all. looks ok on my tablet and phone,  i think it's probably the best bet for tables which 'want to be' too wide.

what do you think?

Reply:
Thanks for testing this.  I still think this could cause a problem if people have ridiculously large values in a field and it is not of uniform length.  For example imagine many rows but one cell has tons of text.  I have seen this happen allot so thats why I think we need to safeguard things with an overflow?

Reply:
it still works, i've updated the blog post with extra markdown and df tables with one abnormally long value.

the nowrap and max-width mean you don't have to protect yourself from overflows, they won't happen. 

in the markdown table you might get a row which is higher than the rest and therefore not so visually attractive but in my mind that's better than clipping and losing data to keep the rows the same height.

the df is truncating the very long value and keeping the row height consistent when it renders the table as pandas always does so that's even less of a problem.

if you're still not keen after seeing how it looks i'll withdraw the pr.

Reply:
have you had a chance to take another look? the difference from the current style is now basically:
```
table {
  white-space: normal;
  max-width: 100%;
  td{
    max-width: none;
  }
}
```
it deals with non-uniform large values as per your concern. shall i resubmit it like this or close it?